tests/metrics/test_pcr_metrics.py

```diff
@@ -0,0 +1,39 @@
+from tests.common import *
+import pandas as pd
+
+
+def test_pc_regression(adata):
+    scIB.me.pc_regression(adata.X, adata.obs["batch"])
+
+
+def test_pcr_batch(adata):
+    # no PCA precomputed
+    score = scIB.me.pcr_comparison(
+        adata, adata,
+        variable='batch',
+        n_comps=50,
+        scale=True
+    )
+    LOGGER.info(f"no PCA precomputed: {score}")
+    assert 0 <= score <= 1 and score < 1e-6
```

Comment:
bit strange this one... `score <= 1` and `score < 1e-6`... first one seems redundant.